src/commands/create.js

```diff
@@ -92,7 +92,7 @@ export default asyncCommand({
 			build: 'preact build',
 			serve: 'preact build && preact serve',
 			dev: 'preact watch',
-			test: 'eslint src && preact test'
+			test: 'eslint src && echo "Error: no test specified" && exit 1'
```

Comment:
This one we actually probably need to keep as `preact test` - that command isn't available yet, but it will just spawn a preconfigured jest. I wanted to get the setup into generated package.json's since we can't retroactively update them later. 

Would you be all right removing this one line? I think we can get the `test` command released really soon.